project/Dependencies.scala

```diff
@@ -734,6 +735,7 @@ object Dependencies {
 
   val `persistence-cassandra-javadsl` = libraryDependencies ++= Seq(
     junit % Test,
+    "javax.annotation" % "jsr250-api" % "1.0",
```

Comment:
Why are these dependencies necessary ?

Reply:
Doesn't compile otherwise (the project names are in the commit messages).  To do with Java's module system.